src/RecordData/Logger.cpp

```diff
@@ -74,7 +74,7 @@ void Logger::recordLogData(LogType type, const char *data, Dest dest) {
 }
 
 // Records log data with a specific timestamp, type, and destination
-void Logger::recordLogData(double timeStamp, LogType type, const char *data, Dest dest = BOTH) {
+void Logger::recordLogData(double timeStamp, LogType type, const char *data, Dest dest) {
```

Comment:
The default here is redundant because it's already given a specified default in the .h file. Having two default declarations generated a warning so I removed it here.